docs/guides/math.server.mdx

```diff
@@ -79,6 +79,10 @@ async function main() {
   ```
 </Note>
 
+<Note>
+  **Note:** If you have a use case that requires dynamic JS expressions inside of math environments (e.g. accessing props, making calculations, etc) you may want to check out [remark-mdx-math-enhanced](https://github.com/goodproblems/remark-mdx-math-enhanced) 
```

Comment:
```suggestion
  **Note:** see also
  [`remark-mdx-math-enhanced`](https://github.com/goodproblems/remark-mdx-math-enhanced),
  which you can use to support JavaScript expressions inside of math (such as to
  access props or to make calculations)
```

How about this one? Notable changes: `e.g.` -> such as, code for project name, lowercase first word, full `JavaScript`, line wrapping